.github/workflows/build-docker.yaml

```diff
@@ -32,6 +38,11 @@ jobs:
           ref: ${{ inputs.ref }}
           submodules: recursive
 
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+        with:
+          platforms: linux/amd64,linux/arm64
```

Comment:
Does this use qemu for emulation?  How slow is it?

Reply:
Yes it does, and it's quite slow, took 1h on my gh action. Best would be to setup a matrix step on an external runner like you did on other actions but I can't setup this myself.

Reply:
I don't think I want to add 1h of emulated Docker builds to every PR.  Let me try to set it up through our self-hosted runners, at least until GitHub starts offering Arm Linux runners publicly.